store/admin.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib import admin
+from ordered_model.admin import OrderedModelAdmin, OrderedTabularInline
+from . import models
+
+
+@admin.register(models.Category)
+class CategoryAdmin(admin.ModelAdmin):
+    list_display = ["name","name_ar"]
```

Comment:
@m7mdGNo add number of products to the list and add a url to view products of that category (list change of the products filtered with category id)